svf-llvm/include/SVF-LLVM/LLVMModule.h

```diff
@@ -69,6 +69,7 @@ class LLVMModuleSet
     std::unique_ptr<LLVMContext> cxts;
     std::vector<std::unique_ptr<Module>> owned_modules;
     std::vector<std::reference_wrapper<Module>> modules;
+    OrderedSet<std::string> usedExtFuncNames;
```

Comment:
remove this and use LLVMFun2SVFFunMap to determine a valid external API function